### ISSUE TITLE
Asterinas fails to boot

### ISSUE THREAD

**Issue by grief8** [asterinas/asterinas#1711]
The following bug occasionally occurs in the past. However, I find it happens much more frequently today. Most of the benchmark [workflow](https://github.com/asterinas/asterinas/actions/runs/12322227950/job/34395354075) cases fails because of it. Does anyone have an idea about it?
```
!!!! X64 Exception Type - 01(#DB - Debug)  CPU Apic ID - 00000000 !!!!
RIP  - 000000000000201F, CS  - 0000000000000038, RFLAGS - 0000000000200002
RAX  - 000000007F9EE010, RCX - 0000000000000001, RDX - 0000000000000002
RBX  - 0000000000000006, RSP - 00000000001CFC9A, RBP - 0000000000000000
RSI  - 0000000000000000, RDI - 00000000FFFFFFFF
R8   - 0000000000000006, R9  - 000000007FF0FFA8, R10 - 0000000000000000
R11  - 00000000000003EE, R12 - 0000000000000001, R13 - 0000000000000002
R14  - 00000000FFFFFFFF, R15 - 000000007E447C08
DS   - 0000000000000030, ES  - 0000000000000030, FS  - 0000000000000030
GS   - 0000000000000030, SS  - 0000000000000030
CR0  - 0000000080010033, CR2 - 0000000000000000, CR3 - 000000007FC01000
CR4  - 0000000000000668, CR8 - 0000000000000000
DR0  - 0000000000000000, DR1 - 0000000000000000, DR2 - 0000000000000000
DR3  - 0000000000000000, DR6 - 00000000FFFF0FF0, DR7 - 0000000000000400
GDTR - 000000007F9DE000 0000000000000047, LDTR - 0000000000000000
IDTR - 000000007F470018 0000000000000FFF,   TR - 0000000000000000
FXSAVE_STATE - 00000000001CF8F0
!!!! Can't find image information. !!!!
```

**Comment by junyang-zh** [comment]
#1710 ?

**Comment by lrh2000** [comment]
> #1710 ?

Yes. It is the issue described in #1710. Thanks @junyang-zh for the link!

**Comment by grief8** [comment]
Closed as the discussion in #1710 .
